carmen-salas/short-response.md

```diff
@@ -0,0 +1,79 @@
+# Problem Set 2.2 - Variables and Control Flow
+## Short Response Questions
+
+1. **What does the code below log? Why?**
+   ```javascript
+    {
+      let singer = 'Omar Apollo';
+    }
+
+    console.log(`My favorite singer is ${singer});
+   ```
+* This will log ‘My favorite singer is Omar Apollo’ because the variable is declared with the string ‘Omar Apollo’ and is being concatenated with the string “my favorite singer is”
+
+2. **What happens when we run the following code? Why?**
+   ```javascript
+const favorite = 'Juan Pablo';
+console.log(`Our favorite Marcy Lab family member is ${favorite}!`);
+favorite = 'Maya';
+console.log(`Actually, ${favorite} is my favorite.`);
+   ```
+*It will log `Our favorite Marcy Lab family member is Juan Pablo!` because the varible ‘favorite’ is declared as a const variable which means it can not be changed once it is declared even if it is initiated again.  
+
+3. **What does the following code log? Why?**
+   ```javascript
+   let score = 90; 
+   let newScore = score;
+   score = 100; 
+   console.log(score, newScore);
+   ```
+*This will log ‘100, score’ because even if the variable ‘score’ was declared with a value of the number 90 it was initiated again with the number 100. The variable ‘newScore’ was declared with a value of the string ‘score’. 
```

Comment:
Run this in the console.

The console will never log the name of the variable. It will log the value it references. Thus, what is the value of `score` at line 28?

Last thing, you said "initiated again with...". The term you are looking for is _reassigned_. The variable was _reassigned_.